### PR TITLE
Privacypolicymodal

### PR DESCRIPTION
**Pull request by johanehr**

### CODE REVIEW

File: src/app/page.tsx
Comment (johanehr) [inline src/app/page.tsx:49]: I think this might be a bit wordy to fit nicely, but if it looks fine, why not! (Haven't checked out the code)

**Review by johanehr — COMMENTED** [review]
Seems like linting is required for the Vercel pre-build to work, BTW

File: src/app/page.tsx
Comment (frankosgit) [reply src/app/page.tsx:49]: Agree!

File: src/app/about/page.tsx
Comment (frankosgit) [reply src/app/about/page.tsx:47]: Great! I couldn't remember what we decided but I think that would be optimal, I will update the text to English now.

File: src/app/about/page.tsx
Comment (frankosgit) [inline src/app/about/page.tsx:60]: or the?

File: src/app/about/page.tsx
Comment (frankosgit) [reply src/app/about/page.tsx:49]: good idea

File: src/app/about/page.tsx
Comment (johanehr) [reply src/app/about/page.tsx:60]: "The app is built..." perhaps, so strong preference!

File: src/app/about/page.tsx
Comment (johanehr) [inline src/app/about/page.tsx:67]: mailto: link?

File: src/app/about/page.tsx
Comment (johanehr) [inline src/app/about/page.tsx:42]: Still in Swedish

Comment (frankosgit) [reply src/app/about/page.tsx:42]: I think we can get rid of this paragraph, thoughts?

Comment (johanehr) [reply src/app/about/page.tsx:42]: Sure!

**Review by johanehr — COMMENTED** [review]
Since I created the PR to see the diff, I can't approve it myself - so I'll let you have the honors, @frankosgit  ;)

**Review by frankosgit — APPROVED** [review]
Great!

**Pull request merged**
